base/applications/clipbrd/clipbrd.c

```diff
@@ -345,6 +345,12 @@ static LRESULT WINAPI MainWndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lP
 {
     switch(uMsg)
     {
+        // For now, the Help dialog item is disabled because of lacking of HTML support
+        case WM_INITDIALOG:
+        {
+            HMENU hMenu = GetMenu(hWnd);
+            EnableMenuItem(GetSubMenu(hMenu, 2), CMD_HELP, MF_BYCOMMAND | MF_GRAYED);
+        }
```

Comment:
You have no `break;` or `return` statement at the end of this `case`, which means it will fall-through into the WM_CREATE code.

Reply:
Oh noes, I must be dumb really... :cry:

Will shall fix it ASAP!